### PR TITLE
chore: update dependency with security issue

### PR DESCRIPTION
**Pull request by kieckhafer**
`dot-prop@4.2.0`, which is installed into `tus-node-server` by `configstore`, shows a medium severity security flag in our snyk tests.

This PR updates `configstore` to v5, which includes an update of `dot-prop`.
image

### CODE REVIEW

**Comment by Acconut** [comment]
Thank you. Can you please also remove Node.js 6 from the Travis CI setup, as Node.js 6 is not supported after your PR: https://travis-ci.org/github/tus/tus-node-server/jobs/675997084

**Comment by kieckhafer** [comment]
@Acconut updated to remove node6

**Comment by Acconut** [comment]
Thank you!

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.0"
```
Comment (kvz) [inline .travis.yml:4]: Does this mean next release is gonna be a major, or is this actually fine to skip over?

Comment (Acconut) [reply .travis.yml:4]: Should probably be a major release, yes.

**Pull request merged**